core/dbt/config/runtime.py

```diff
@@ -30,6 +31,21 @@
 from dbt.dataclass_schema import ValidationError
 
 
+def load_project(
+    project_root: str,
+    profile: HasCredentials,
+    cli_vars: Optional[Dict[str, Any]] = None,
+) -> Project:
+    # get the project with all of the provided information
+    version_check = bool(flags.VERSION_CHECK)
```

Comment:
should we be getting `VERSION_CHECK` from the [click params](https://github.com/dbt-labs/dbt-core/blob/main/core/dbt/cli/params.py#L361) at this point?

Reply:
Updated to pass `VERSION_CHECK` as a param